libs/filameshio/include/filameshio/filamesh.h

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2018 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef TNT_FILAMENT_FILAMESHIO_FILAMESH_H
+#define TNT_FILAMENT_FILAMESHIO_FILAMESH_H
+
+#include <filament/Box.h>
+
+namespace filamesh {
```

Comment:
Is this header only for testing? We shouldn't need to make this public

Reply:
The header is used by `libs/filameshio` (the reader) and `tools/filamesh` (the writer), and `test_filamesh`.  It does not need be installed so it is not included in DIST_HDRS.

Reply:
Makes sense, thanks